_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer.cpp

```diff
@@ -711,17 +702,21 @@ void PackerVA::PackQmatrix(const H265Slice *pSlice)
 
 void PackerVA::PackAU(const H265DecoderFrame *frame, TaskSupplier_H265 * supplier)
 {
-    H265DecoderFrameInfo * sliceInfo = frame->m_pSlicesInfo;
-    int sliceCount = sliceInfo->GetSliceCount();
+    if (!frame)
```

Comment:
these two can be combined in one condition

Reply:
Fixed